app/org/sagebionetworks/bridge/services/ParticipantOptionsService.java

```diff
@@ -1,64 +1,90 @@
 package org.sagebionetworks.bridge.services;
 
 import java.util.Map;
+import java.util.Set;
 
 import org.sagebionetworks.bridge.dao.ParticipantOption;
 import org.sagebionetworks.bridge.dao.ParticipantOption.SharingScope;
 import org.sagebionetworks.bridge.dynamodb.OptionLookup;
+import org.sagebionetworks.bridge.models.accounts.DataGroups;
+import org.sagebionetworks.bridge.models.accounts.ExternalIdentifier;
 import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
 
+/**
+ * Provides type-safe access to all the options that can be persisted for a health code, as well as validation. 
+ * The values are saved as strings and can be exported as such (in spreadsheets, Synapse, and so forth).
+ */
 public interface ParticipantOptionsService {
```

Comment:
I feel like this approach isn't scalable. This approach means that for every option, we'll need to add a new getter/setter pair in ParticipantOptionsService. I feel like it would be more scalable to have get/setOptionAsBoolean(), get/setOptionAsEnum() (which takes the Enum class as a parameter), get/setOptionAsString(), get/setOptionAsStringSet(), etc.